.packit.yaml

```diff
@@ -0,0 +1,47 @@
+# https://packit.dev/docs/configuration/
+
+specfile_path: fido-device-onboard.spec
+
+files_to_sync:
+    - fido-device-onboard.spec
+    - .packit.yaml
+
+upstream_package_name: fido-device-onboard
+downstream_package_name: fido-device-onboard
+
+upstream_tag_template: v{version}
+copy_upstream_release_description: true
+
+srpm_build_deps:
+- cargo
+
+actions:
+  create-archive:
+  - "cargo vendor vendor"
+  - bash -c "git archive --prefix=fido-device-onboard-rs-${PACKIT_PROJECT_VERSION}/ --format=tar HEAD > fido-device-onboard-rs-${PACKIT_PROJECT_VERSION}.tar"
+  - bash -c "tar -xvf fido-device-onboard-rs-${PACKIT_PROJECT_VERSION}.tar"
+  - bash -c "cp -Ra vendor fido-device-onboard-rs-${PACKIT_PROJECT_VERSION}"
+  - bash -c "tar -czf fido-device-onboard-rs-${PACKIT_PROJECT_VERSION}.tar.gz fido-device-onboard-rs-${PACKIT_PROJECT_VERSION}"
+  - bash -c "rm -rf fido-device-onboard-rs-${PACKIT_PROJECT_VERSION} fido-device-onboard-rs-${PACKIT_PROJECT_VERSION}.tar vendor"
+  - bash -c "ls -1 ./fido-device-onboard-rs-*.tar.gz"
+  fix-spec-file:
+  - sed -i fido-device-onboard.spec -e "s/with_packit 0/with_packit 1/"
+
+jobs:
+- job: copr_build
+  trigger: pull_request
+  targets:
+  - centos-stream-9-aarch64
+  - centos-stream-9-x86_64
+  - fedora-development-aarch64
+  - fedora-development
+- job: copr_build
+  trigger: commit
+  branch: main
+  owner: "runcom" # copr repo namespace
```

Comment:
TODO: change this, @nullr0ute or @miabbott might be able to help creating a fedora-iot or edge namespace

Reply:
Looking at it now

Reply:
I've removed this piece for now until we have the copr group and the PR is ready so other PR can rebase and have their PR create RPMs too

Reply:
there's probably a shorter URL